data/dev/dataset.csv

```diff
@@ -7,7 +7,7 @@ Next generation sequencing was performed on the libraries using an Illumina GAII
 144,163,147,100094,Data and software to accompany the paper: Applying compressed sensing to genome-wide association studies.,"The aim of a genome-wide association study (GWAS) is to isolate DNA markers for variants affecting phenotypes of interest. Linear regression is employed for this purpose, and in recent years a signal-processing paradigm known as compressed sensing (CS) has coalesced around a particular class of regression techniques. CS is not a method in its own right, but rather a body of theory regarding signal recovery when the number of predictor variables (i.e., genotyped markers) exceeds the sample size. The paper shows the applicability of compressed sensing (CS) theory to genome-wide association studies (GWAS), where the purpose is to ﬁnd trait-associated tagging markers (genetic variants). Analysis scripts are contained in the compressed CS file. Mock data and scripts are found in the compressed GD file. The example scripts found in the CS repository require the GD files to be unpacked in a separate folder. Please look at accompanying readme pdfs for both repositories and annotations in the example scripts before using.",171860992,ftp://parrot.genomics.cn/gigadb/pub/10.5524/100001_101000/100094,Published,,,2014-06-06,,1,AWLGPriZKd41XEaS,,,,
-200,336,212,100142,"Supporting scripts and data for ""Investigation into the annotation of protocol sequencing steps in the Sequence Read Archive"".","The workflow for the production of high-throughput sequencing data from nucleic acid samples is complex. There are a series of protocol steps to be followed in the preparation of samples for next-generation sequencing.  The quantification of bias in a number of protocol steps, namely DNA fractionation, blunting, phosphorylation, adapter ligation and library enrichment, remains to be determined. We examined the experimental metadata of the public repository Sequence Read Archive (SRA) in order to ascertain the level of annotation of important sequencing steps in submissions to the database. ",384263168,ftp://parrot.genomics.cn/gigadb/pub/10.5524/100001_101000/100142/,Published,100142newversion.xls,a98f1c13a46501707fb3fba270deb6e3,2015-05-11,,3,eG8EFsRibZQzrtNw,,,,
```

Comment:
I think this change to remove a space character will decrease the size of the readme_100142.txt file from 1997 to 1996 bytes. When I manually test this PR by running `./postUpload.sh 100142`, it will create readme_100142.txt and upload into dev Wasabi directory. This will then break the `UpdateFileSizeCest` files metadata console functional test.

Reply:
The file `readme_100142.txt` in the wasabi bucket has been updated with file size `1996` now.